index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="tool-box">
+        <div class="headder">
+            <h1><a class="name" href="#">Welcome</a></h1>
```

Comment:
- [ ] As required by the previous reviewer you should just change this to an `a` tag, you don't need to put it inside `hx`. Also, the HTML file uses only one `h1` and you already have it 😁 😁